astropy/io/fits/hdu/hdulist.py

```diff
@@ -17,13 +16,21 @@
 from .image import PrimaryHDU, ImageHDU
 from astropy.io.fits.file import _File, FILE_MODES
 from astropy.io.fits.header import _pad_length
-from astropy.io.fits.util import (_is_int, _tmp_name, fileobj_closed, ignore_sigint,
-                    _get_array_mmap, _free_space_check, fileobj_mode, isfile)
+from astropy.io.fits.util import (_is_int, _tmp_name, fileobj_closed,
+                                  ignore_sigint, _get_array_mmap,
+                                  _free_space_check, fileobj_mode, isfile)
```

Comment:
Since you already changed these lines maybe this is a good opportunity to list the functions alphabetically?